client/src/components/LogChannel.ts

```diff
@@ -4,14 +4,14 @@ import { OutputChannel, l10n, window } from "vscode";
 
 import { appendLog } from "./LogViewer";
 
-let outputChannel: OutputChannel;
+let log: OutputChannel;
+if (!log) {
+  log = window.createOutputChannel(l10n.t("SAS Log"), "sas-log");
+}
+export { log };
```

Comment:
how about subscribe the store inside this component and not necessary to export it

Reply:
Fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/0ce419ea20fce295cf620a7f6b83f9a0c0ff9104